frontend/src/components/cart-page/CartSummarySection.tsx

```diff
@@ -0,0 +1,72 @@
+import { useContext, useMemo } from 'react'
+import { useNavigate } from 'react-router-dom';
+
+import { ShoppingCartContext } from '../../contexts/shopping-cart-context/shoppingCartContext';
+import { Box, Button, Card, CardContent, Divider, Typography } from "@mui/material";
+
+import cartStyles from './cartStyles';
+const {
+    totalCardStyle,
+    buttonContainerStyle,
+    buttonStyle,
+} = cartStyles;
```

Comment:
Not sure how useful it is to export cartStyles as an object, i think it's cleaner to import what you need like:
`import  {
    totalCardStyle,
    buttonContainerStyle,
    buttonStyle,
} from './cartStyles;`